### ISSUE TITLE
Is there a way to default to keep forever? I hate having to change it every time, and frequently forget to, making links die.

### ISSUE THREAD

**Issue by levihb** [bepasty/bepasty-server#282]
I use this to share online, and I would generally prefer it to be up forever, especially when I share in forums etc. But sometimes I forget to change it from one month to keep forever.

And also it's just annoying having to change it. Is there some way to just default to forever?

**Comment by ThomasWaldmann** [comment]
https://github.com/bepasty/bepasty-server/blob/master/src/bepasty/utils/date_funcs.py#L8

That's the place in the source - can you make a PR to make this configurable?

Back then, when I implemented that, I intentionally did not set "forever" as default to get a self-cleaning-by-default behaviour.

**Comment by levihb** [comment]
> https://github.com/bepasty/bepasty-server/blob/master/src/bepasty/utils/date_funcs.py#L8
> 
> That's the place in the source - can you make a PR to make this configurable?
> 
> Back then, when I implemented that, I intentionally did not set "forever" as default to get a self-cleaning-by-default behaviour.

Ahh cheers. Sure I'll try and get around to it on Sunday. Do you have any specific way you'd like it configurable? Maybe a setting with a value of `MINUTES,30` to set it to default to 30 mins, etc?

**Comment by ThomasWaldmann** [comment]
Yes, for example. See other places that deal with configured values.

**Comment by ThomasWaldmann** [comment]
@levihb do you want to make a PR for this?
